src/test/java/com/scottlogic/filter/FilterContentKeywordTest.java

```diff
@@ -0,0 +1,43 @@
+package com.scottlogic.filter;
+
+import com.scottlogic.UserPost;
+import com.scottlogic.utils.UserPostFilterData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class FilterContentKeywordTest {
+
+    FilterContentKeyword filterContentKeyword = new FilterContentKeyword();
+
+    @Test
+    @DisplayName("Functionality with 'you'")
+    void setFilterContentKeyword_you() {
+        List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
+        List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_you();
+        Assertions.assertEquals(expectedFilteredList, filterContentKeyword.filter(postsToFilter, "you"));
+    }
+
+    @Test
+    @DisplayName("Functionality with 'You'")
+    void setFilterContentKeyword_You() {
+        List<UserPost> postsToFilter = UserPostFilterData.getUnfilteredData();
+        List<UserPost> expectedFilteredList = UserPostFilterData.getFilteredContentKeyword_You();
+        Assertions.assertEquals(expectedFilteredList, filterContentKeyword.filter(postsToFilter, "You"));
```

Comment:
Does this filter support sentences as input?
How would whitespace and special characters affect this?
Could you treat a number as a keyword?

Try some expected failure cases,

like "yuo" would return an empty list.

Reply:
Inserted new tests for 'yuo' (return empty list), 'Funky funky' (returned one userpost - due to capitalisation) and number(1) and special character (?) and they work as expected!  

As an extension (or maybe even now) I may use the asterisk notation to denote everything before/after - what do you think? Worth my time?